osu.Game.Tournament.Tests/TournamentTestScene.cs

```diff
@@ -123,11 +121,11 @@ public virtual void SetUpSteps()
                     },
                     Players =
                     {
-                        new APIUser { Username = "Hello", Statistics = new UserStatistics { GlobalRank = 12 } },
-                        new APIUser { Username = "Hello", Statistics = new UserStatistics { GlobalRank = 16 } },
-                        new APIUser { Username = "Hello", Statistics = new UserStatistics { GlobalRank = 20 } },
-                        new APIUser { Username = "Hello", Statistics = new UserStatistics { GlobalRank = 24 } },
-                        new APIUser { Username = "Hello", Statistics = new UserStatistics { GlobalRank = 30 } },
+                        new TournamentPlayer { Username = "Hello", Rank = 12 },
```

Comment:
I think we should call this `TournamentUser`. The `Player` term is already used by the main gameplay screen and I can 100% foresee a `TournamentPlayer` subclass existing in the future..

Reply:
`TournamentUser` sounded off at first since players aren't... well, "users". But the point about `TournamentPlayer` is pretty fair, will change.

Reply:
They are `IUser` so I'm not sure I'd call them "not users".

Reply:
In code, yeah, but was thinking about this in words (i.e. reading it as "tournament user", which would usually sound like a user of the tournament).